### ISSUE TITLE
Can't open! Traceback included.

### ISSUE THREAD

**Issue by ryanbugden** [Typefounding/underlineStrikethrough#7]
Hi there! Love this extension. I've been getting this traceback when trying to open it. Are you getting that in RF4? I think it has something to do with the collection of the glyphs for the preview.

```
Traceback (most recent call last):
  File "Underline StrikeThrough.py", line 213, in <module>
  File "lib/scripting/nameSpaceCallbacks.pyc", line 51, in OpenWindow
  File "Underline StrikeThrough.py", line 98, in __init__
  File "Underline StrikeThrough.py", line 128, in setUI
  File "/Applications/RoboFont-4_2b2.app/Contents/Resources/lib/python3.7/mojo/UI.py", line 1029, in set
  File "lib/UI/spaceCenter/multiLineScrollScrollView.pyc", line 569, in setGlyphRecords_
  File "lib/UI/spaceCenter/multiLineScrollScrollView.pyc", line 592, in setKerningInGlyphRecords_
  File "/Applications/RoboFont-4_2b2.app/Contents/Resources/lib/python3.7/defcon/objects/kerning.py", line 103, in find
  File "/Applications/RoboFont-4_2b2.app/Contents/Resources/lib/python3.7/fontTools/ufoLib/kerning.py", line 63, in lookupKerningValue
AttributeError: 'NoneType' object has no attribute 'startswith'
```

**Comment by benkiel** [comment]
Ryan, will look into, can't promise a timeline... That view needs to get updated with merz

**Comment by ryanbugden** [comment]
No rush, thanks Ben!

**Comment by benkiel** [comment]
Oh! I see the issue from what you fixed; the margin glyph isn't a real glyph and when the line view tries to do kerning, it gives up

**Comment by benkiel** [comment]
Fixed by #10
